Rakefile

```diff
@@ -110,7 +110,8 @@ end
 desc "Validate docs"
 task :validate_docs do
   print_info "Checking docs are up to date"
-  sh "sourcekitten doc --module-name SourceryRuntime > docs.json && bundle exec jazzy --skip-undocumented --no-download-badge && rm docs.json"
+  sh "sourcekitten doc --module-name SourceryRuntime -- -workspace Sourcery.xcworkspace -scheme Sourcery-Release -derivedDataPath #{BUILD_DIR}tmp/ > docs.json && bundle exec jazzy --skip-undocumented --no-download-badge && rm docs.json"
+  sh "rm -fr #{BUILD_DIR}tmp/"
```

Comment:
Does it make sense to create a rake task `clean_temp` and run it after `build` and `validate_docs` tasks?

Something like:
```
task : validate_docs => [:clean_temp] do
...
end
```

Reply:
I am not a `Rakefile` specialist but I do not think that would work.
What you describe here is a prerequisite (meaning it will run _before_ the task), where we want a "post" step.

I find it easier to understand to keep related commands close to each other.
In our case here, we put files in the `tmp/` folder and remove them right after. It makes the task _self contained_.

Reply:
> In our case here, we put files in the tmp/ folder and remove them right after

I think it's a good point, maybe then I would only declare a variable `#{BUILD_DIR}tmp/`. Other than that - good job 👍

Reply:
I've added the variable in 5cfd48b 😉